HelpersGetPackageParameters.md

```diff
@@ -77,8 +77,9 @@ if (!$pp['LICENSE']) { $pp['LICENSE'] = '1234' }
 ~~~powershell
 
 $pp = Get-PackageParameters
+if (!$pp['UserName']) { $pp['UserName'] = "$env:UserName" }
 # Requires 0.10.8 for Read-Host -AsSecureString
-if (!$pp['Password']) { $pp['Password'] = Read-Host "Enter password for $userName:" -AsSecureString}

```

Comment:
If you are seeing that, maybe it should be set as `$userName :` to separate the variable from the colon. The change however is accurate. 👍